libs/pilight/protocols/GPIO/ds18b20.c

```diff
@@ -133,6 +138,17 @@ static void *ds18b20Parse(void *param) {
 								}
 								memset(content, '\0', bytes+1);
 
+                                //try to set resolution here - set it temporary to not wear out the EEEPROM
```

Comment:
`// Try to set resolution here - set it temporary to not wear out the EEPROM`

(grammar/spelling)